dashboard/app/views/layouts/_header.html.haml

```diff
@@ -107,7 +107,7 @@
   - user_progress = current_user == nil ? 'null' : summarize_user_progress(script, view_as, view_as == current_user && @public_caching).to_json
 
   - if view_as
-    - stage_extras_enabled = view_as.stage_extras_enabled?(script)
+    - stage_extras_enabled = view_as.lesson_extras_enabled?(script)
```

Comment:
should this be `- lesson_extras_enabled = view_as.lesson_extras_enabled?(script)` ?

Reply:
Good catch, fixed